package.json

```diff
@@ -18,20 +18,20 @@
     "build-plugin-js": "npm run build:prod && cat build/static/runtime*.js build/static/polyfills*.js build/static/main*.js >build/static/perspective_authorship_widget.js"
   },
   "peerDependencies": {
-    "@angular/animations": "6.0.0",
-    "@angular/cdk": "6.0.1",
-    "@angular/common": "6.0.0",
-    "@angular/compiler": "6.0.0",
-    "@angular/compiler-cli": "6.0.0",
-    "@angular/core": "6.0.0",
-    "@angular/elements": "github:angular/elements-builds#labs/elements",
-    "@angular/forms": "6.0.0",
-    "@angular/http": "6.0.0",
-    "@angular/material": "6.0.1",
-    "@angular/platform-browser": "6.0.0",
-    "@angular/platform-browser-dynamic": "6.0.0",
-    "@angular/router": "6.0.0",
-    "gsap": "1.20.4",
+    "@angular/animations": "^6.0.0",
+    "@angular/cdk": "^6.0.1",
+    "@angular/common": "^6.0.0",
+    "@angular/compiler": "^6.0.0",
+    "@angular/compiler-cli": "^6.0.0",
+    "@angular/core": "^6.0.0",
+    "@angular/elements": "^6.0.0",
+    "@angular/forms": "^6.0.0",
+    "@angular/http": "^6.0.0",
+    "@angular/material": "^6.0.1",
+    "@angular/platform-browser": "^6.0.0",
+    "@angular/platform-browser-dynamic": "^6.0.0",
+    "@angular/router": "^6.0.0",
+    "gsap": "^1.20.4",
```

Comment:
We can't use semantic versioning for gsap because 1.20.5 has breaking changes: https://github.com/greensock/GreenSock-JS/issues/266

Reply:
Actually, looks like the API changes are trivial to apply, so I'll just do that and then we don't have to make this horribly inflexible in the peer dependencies :)